packages/web-api/src/controllers/scan-report-controller.ts

```diff
@@ -23,16 +23,21 @@ export class ScanReportController extends ApiController {
     }
 
     public async handleRequest(): Promise<void> {
+        const scanId = <string>this.context.bindingData.scanId;
         const reportId = <string>this.context.bindingData.reportId;
+        this.logger.setCommonProperties({ source: 'getScanReportRESTApi', scanId, reportId });
+
         if (!this.guidGenerator.isValidV6Guid(reportId)) {
```

Comment:
Not necessarily important for this pr, but should we also validate of scanId here?

Reply:
No, we do not use scan id directly to retrieve the report file from a blob